sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/ServiceBusEndToEndTests.cs

```diff
@@ -346,6 +366,16 @@ private async Task TestSingleDrainMode<T>(bool sendToQueue)
             }
         }
 
+        private static Action<IHostBuilder> BuildHostWithAutoCompleteDisabled<T>()
+        {
+            return builder =>
+                builder.ConfigureDefaultTestHost<T>(b =>
```

Comment:
This should just be ConfigureWebJobs to avoid adding the logger twice - https://github.com/Azure/azure-sdk-for-net/pull/21360#discussion_r639374412